tests/system/python/conftest.py

```diff
@@ -668,6 +669,33 @@ def _disable_sch(fledge_url, sch_name):
     return _disable_sch
 
 
+@pytest.fixture(scope="function", autouse=True)
+def collect_support_bundle(request):
+    def _collect_support_bundle():
+        PROJECT_ROOT = subprocess.getoutput("git rev-parse --show-toplevel")
+        try:
+            conn = http.client.HTTPConnection("{}".format(url))
+            conn.request("POST", "/fledge/support")
+            r = conn.getresponse()
+            assert 200 == r.status
+            r = r.read().decode()
+            jdoc = json.loads(r)
```

Comment:
def post_request from utils.py can be used instead of this.

Reply:
for that I have to import utils into conftest file